docs/guide/using-vue.md

```diff
@@ -4,15 +4,15 @@
 
 Because VuePress applications are server-rendered in Node.js when generating static builds, any Vue usage must conform to the [universal code requirements](https://ssr.vuejs.org/en/universal.html). In short, make sure to only access Browser / DOM APIs in `beforeMounted` or `mounted` hooks.
 
-If you are using or demoing a component that is not SSR friendly (for example containing custom directives), you can wrap them inside the built-in `<ClientOnly>` component:
+If you are using or demoing components that are not SSR friendly (for example containing custom directives), you can wrap them inside the built-in `<ClientOnly>` component:
 
 ``` md
 <ClientOnly>
   <NonSSRFriendlyComponent/>
 </ClientOnly>
 ```
 
-Note this does not fix components or libraries that access Browser APIs **on import** - in order to use code that assumes a browser environment on import, you need to dynamically import them in proper lifecycle hooks:
```

Comment:
I'd leave this sentence as-is because you can do this in either `beforeMount` or `mounted`.